podman_compose.py

```diff
@@ -762,7 +762,7 @@ async def assert_cnt_nets(compose, cnt):
                 args.extend(("--opt", f"{key}={value}"))
             ipam = net_desc.get("ipam", None) or {}
             ipam_driver = ipam.get("driver", None)
-            if ipam_driver:
+            if ipam_driver and ipam_driver not in ("default"):
```

Comment:
I think the code would be clearer if we compared using `!=`. Also I think there's a bug here, because `("default")` is not a tuple. `("default",)` would be a tuple.

Reply:
You are right, the missing ',' is strange because my tests worked ... 
Yes, we can compare using `!=`.
I do a patch with it.